pkg/util/clusterdata/clusterdata.go

```diff
@@ -72,7 +72,7 @@ func (e *bestEffortEnricher) enrichOne(ctx context.Context, oc *api.OpenShiftClu
 		return
 	}
 
-	// TODO: Get rid of the wrapping RoundTripper once implementation of the KEP below lands into openshift/client-go:
+	// TODO: Get rid of the wrapping RoundTripper once implementation of the KEP below lands into openshift/kubernetes-client-go:
```

Comment:
updated this to avoid confusion. 
`openshift/client-go` is client for `oc` and is not a fork of an upstream

Reply:
@olga-mirensky it should be both, really. In `pkg/util/clusterdata/cluster_version_task.go` we use a client from `openshift/client-go`. In `pkg/util/clusterdata/service_principal_task.go` a client from the fork. In `pkg/util/clusterdata/worker_profiles_task.go` a client from `github.com/openshift/cluster-api` (didn't find anything in any of `client-go`s to get the worker profiles).

I guess that both `client-go` repos will be updated at the same time.